docs/getting-started/istio/index.md

```diff
@@ -33,15 +33,28 @@ spec:
           virtualService:
             # Reference to a VirtualService which the controller updates with canary weights
             name: rollouts-demo-vsvc
+            # Optional if there is a single HTTP route in the VirtualService, otherwise required
             routes:
-            - primary # optional if there is a single route in VirtualService, required otherwise
+            - http-primary
+            # Optional if there is a single HTTPS/TLS route in the VirtualService, otherwise required
+            tlsRoutes:
+            # Below fields are optional but if defined, they should match exactly with at least one of the TLS route match rules in your VirtualService
+            - port: 443 # Only required if you want to match any rule in your VirtualService which contains this port
+              # Only required if you want to match any rule in your VirtualService which contain all these SNI hosts
+              sniHosts:
+              - reviews.bookinfo.com
+              - localhost
 ...
 ```
 
 The VirtualService and route referenced in `trafficRouting.istio.virtualService` is required
-to have a HTTP route which splits between the stable and canary Services, referenced in the rollout.
-In this guide, those Services are named: `rollouts-demo-stable` and `rollouts-demo-canary` 
-respectively. The weight values for these services used should be initially set to 100% stable, 
+to have either an HTTP or a TLS route spec that splits between the stable and the canary services,
```

Comment:
can we have both http and tls route in a rollout spec? Could not infer the answer from the description.

Reply:
Will do! Thanks